config/jobs/kubernetes-sigs/sig-windows/release-1.24-windows.yaml

```diff
@@ -0,0 +1,82 @@
+presets:
+- labels:
+    preset-capz-windows-common-124: "true"
+  env:
+  - name: "KUBERNETES_VERSION"
+    value: "latest-1.24"
+  - name: E2E_ARGS
+    value: "-kubetest.use-ci-artifacts"
+  - name: WINDOWS
+    value: "true"
+  - name: AZURE_NODE_MACHINE_TYPE
+    value: "Standard_D4s_v3"
+periodics:
+- annotations:
+    testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
+    testgrid-dashboards: sig-release-1.24-informing, sig-windows-1.24-release, sig-windows-signal
+    testgrid-tab-name: capz-windows-containerd-1.24
+  decorate: true
+  decoration_config:
+    timeout: 4h0m0s
+  extra_refs:
+  - base_ref: main
+    org: kubernetes-sigs
+    path_alias: sigs.k8s.io/cluster-api-provider-azure
+    repo: cluster-api-provider-azure
+  interval: 3h
+  labels:
+    preset-azure-cred-only: "true"
+    preset-capz-containerd-latest: "true"
+    preset-capz-windows-2019: "true"
+    preset-capz-windows-common-124: "true"
+    preset-capz-windows-parallel: "true"
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  name: ci-kubernetes-e2e-capz-master-containerd-windows-1-24
```

Comment:
nit: can you make `name: ci-kubernetes-e2e-capz-master-containerd-windows-1-24` be the first line for the entry?
It helps when navigating these files and using code-folding.